--- ./configure.in.org	2005-06-02 09:04:48.095242368 +0200
+++ ./configure.in	2005-06-02 09:06:44.450553672 +0200
@@ -97,7 +97,7 @@
 AC_PATH_PROG(UPDATE_MIME_DATABASE, update-mime-database)
 AC_PATH_PROG(UPDATE_DESKTOP_DATABASE, update-desktop-database)
 
-ALL_LINGUAS="ar az bg ca cs de el en_CA en_GB es eu fr he hr hu is it ja ko lt ms nl no pa pl pt pt_BR ru rw sq sr sr@Latn sv tr uk zh_CN zh_TW"
+ALL_LINGUAS="ar az bg ca cs de el en_CA en_GB es eu fr he hr hu is it ja ko lt ms nb nl pa pl pt pt_BR ru rw sq sr sr@Latn sv tr uk zh_CN zh_TW"
 AC_SUBST(ALL_LINGUAS)
 GETTEXT_PACKAGE=gdesklets
 AC_SUBST(GETTEXT_PACKAGE)
