diff -Nuard gDesklets-0.23.orig/data/Makefile.am gDesklets-0.23/data/Makefile.am
--- gDesklets-0.23.orig/data/Makefile.am	2003-10-02 17:54:11.000000000 +0200
+++ gDesklets-0.23/data/Makefile.am	2003-11-07 00:59:21.000000000 +0100
@@ -73,9 +73,11 @@
 
 	$(LN_S) -f $(pixmapdir)/gdesklets.png \
 	$(DESTDIR)$(fakepixmapdir)/gdesklets.png;
-
+	
+if GCONF_SCHEMAS_INSTALL
 	GCONF_CONFIG_SOURCE=$(GCONF_SCHEMA_CONFIG_SOURCE) $(GCONFTOOL) \
 	--makefile-install-rule $(srcdir)/$(schema_DATA); 	
+endif
 
 if USERINST
 	$(PYTHON) desktop-helper.py install;
