diff -urN aaa/gDesklets-0.12/data/Makefile.am gDesklets-0.12/data/Makefile.am
--- aaa/gDesklets-0.12/data/Makefile.am	2003-08-17 17:09:22.000000000 +0200
+++ gDesklets-0.12/data/Makefile.am	2003-08-17 22:00:53.000000000 +0200
@@ -61,7 +61,7 @@
 		chmod 0755 $(DESTDIR)$(fakepixmapdir); \
 	fi
 
-	$(LN_S) -f $(DESTDIR)$(PIXMAPDIR)/gdesklets.png \
+	$(LN_S) -f $(PIXMAPDIR)/gdesklets.png \
 	$(DESTDIR)$(fakepixmapdir)/gdesklets.png;
 
 if USERINST
diff -urN aaa/gDesklets-0.12/Makefile.am gDesklets-0.12/Makefile.am
--- aaa/gDesklets-0.12/Makefile.am	2003-08-17 17:09:22.000000000 +0200
+++ gDesklets-0.12/Makefile.am	2003-08-17 22:00:11.000000000 +0200
@@ -39,4 +39,4 @@
 	fi
 
 	$(INSTALL_SCRIPT) gdesklets $(DESTDIR)${coredir}/gdesklets; \
-	$(LN_S) -f $(DESTDIR)${coredir}/gdesklets $(DESTDIR)$(bindir)/gdesklets;
+	$(LN_S) -f ${coredir}/gdesklets $(DESTDIR)$(bindir)/gdesklets;
