--- gDesklets-0.33/configure.in.wiget	2005-01-03 09:01:44.567506649 +0100
+++ gDesklets-0.33/configure.in	2005-01-03 09:03:01.740162941 +0100
@@ -85,7 +85,7 @@
 AC_SUBST(TILING_CFLAGS)
 AC_SUBST(TILING_LIBS)
 
-ALL_LINGUAS="ar az bg ca cs de el en_CA en_GB es eu fr he hr hu is it ja lt ms nl no pa pl pt pt_BR ru sq sr sr@Latn sv tr ko uk"
+ALL_LINGUAS="ar az bg ca cs de el en_CA en_GB es eu fr he hr hu is it ja lt ms nb nl pa pl pt pt_BR ru sq sr sr@Latn sv tr ko uk"
 AC_SUBST(ALL_LINGUAS)
 GETTEXT_PACKAGE=gdesklets
 AC_SUBST(GETTEXT_PACKAGE)
