diff -urN gDesklets-0.26.2.orig/libdesklets/Disk.py gDesklets-0.26.2/libdesklets/Disk.py
--- gDesklets-0.26.2.orig/libdesklets/Disk.py	2004-08-13 21:56:22.113232392 +0200
+++ gDesklets-0.26.2/libdesklets/Disk.py	2004-08-13 21:56:51.983691392 +0200
@@ -1,6 +1,6 @@
 import glibtop
 import polling
-
+import os
 
 class Disk:
 
@@ -43,15 +43,20 @@
     #:/function
     #
     def __poll_size(self, partition):
-
-        blocks, bfree, bavail, files, ffree = glibtop.get_fsusage(partition)
-        free = float(bfree * 512)
-        total = float(blocks * 512)
-        used = total - free
-
-        return (total, used)#, free)
-
-
+                                                                                                                                
+          try:
+              import statvfs
+              statvfs_data = os.statvfs(partition)
+              free = float(statvfs_data[statvfs.F_BFREE] * statvfs_data[statvfs.F_FRSIZE])
+              total = float(statvfs_data[statvfs.F_BLOCKS] * statvfs_data[statvfs.F_FRSIZE])
+          except:
+              blocks, bfree, bavail, files, ffree = glibtop.get_fsusage(partition)
+              free = float(bfree * 512)
+              total = float(blocks * 512)
+                                                                                                                                
+          used = total - free
+                                                                                                                                
+          return (total, used)#, free)
 
     #
     #:function get_fs | partition | Returns the filesystem for the given
