--- gDesklets-0.33/configure.in.wiget	2005-01-01 18:04:48.000000000 +0100
+++ gDesklets-0.33/configure.in	2005-01-03 08:59:46.739872833 +0100
@@ -45,7 +45,7 @@
 fi
 
 AM_PATH_PYTHON([2.3])
-PYTHON_PREFIX=`python -c "import sys; print sys.prefix"`
+PYTHON_PREFIX=`$PYTHON -c "import sys; print sys.prefix"`
 PYTHON_CFLAGS="-I$PYTHON_PREFIX/include/python$PYTHON_VERSION"
 AC_CHECK_HEADER([$PYTHON_PREFIX/include/python$PYTHON_VERSION/Python.h],,
     AC_ERROR([Can't find Python.h! You will need the python development package
