diff -urN aaa/gDesklets-0.10/locale/install_locales.py gDesklets-0.10/locale/install_locales.py
--- aaa/gDesklets-0.10/locale/install_locales.py	2003-08-10 12:30:13.000000000 +0200
+++ gDesklets-0.10/locale/install_locales.py	2003-08-11 17:09:08.000000000 +0200
@@ -26,8 +26,7 @@
     print
 
 
-potfile = os.path.basename(commands.getoutput("ls ../po/*.pot"))
-domain, nil = os.path.splitext(potfile)
+domain = "gdesklets"
 print domain
 files = os.listdir("../po")
 for f in files:
